docs/reference/troubleshooting/discovery-issues.asciidoc

```diff
@@ -118,3 +119,79 @@ include::network-timeouts.asciidoc[tag=troubleshooting-network-timeouts-threads]
 If a node joins the cluster but {es} determines it to be faulty then it will be
 removed from the cluster again. See <<cluster-fault-detection-troubleshooting>>
 for more information.
+
+[discrete]
+[[discovery-master-split-brain]]
+=== Multiple masters are elected
+
+Infrequently, {es} can enter a {wikipedia}/Split-brain_(computing)[split-brain] 
+state, where nodes within a cluster disagree on which node is elected 
+as master node. This is a non-viable state for the cluster, and can lead to 
+inconsistencies and data loss. 
+
+To prevent state from occurring, we recommend the following practices:
+
+* <<modules-discovery-voting,Avoid even 
+master-eligible node counts>> by creating an odd number of master-eligible nodes or adding a voting-only <<high-availability-cluster-design-two-nodes, tiebreaker>>
+
+* <<modules-discovery-voting,Avoid stopping more half of the voting nodes>> at the same time
+
+* Remove the <<initial_master_nodes,`cluster.initial_master_nodes` 
+setting>> after <<modules-discovery-bootstrap-cluster,bootstrapping a cluster>>
+
+To mitigate the impact of split-brain, we recommend having a <<high-availability,highly 
```

Comment:
how does high availability help with this? the fallback cluster? the designing for resilience topic is pretty huge and it's unclear how it connects, and you mention snapshots explicitly.

Reply:
1. HA helps have your data across multiple nodes so that you could just deprecate the problematic nodes instead of using the [detach-cluster tool](https://www.elastic.co/guide/en/elasticsearch/reference/master/node-tool.html#node-tool-detach-cluster) which jeopardizes data.
2. Support's default stance when we encounter split brain is to check if customer has snapshots to restore into a new cluster as best resolution path.